### PR TITLE
Adding in migration squashing template

### PR DESCRIPTION
**Pull request by BKorenek**
This is a PR to add a template for migration squashing.

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/Standard-Migration-Squashing-Template.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+# Description
+We have 10 or more migration files
+
+## Negative Impact:
+This causes QA script, and general project initialization to slow down costing us developer hours
+
+# Proposed Solution:
+Squash migrations down into the init_schema directory of the migrations directory. 
+Create new migration classes as necessary.
+
```
Comment (rashidsaf) [inline .github/ISSUE_TEMPLATE/Standard-Migration-Squashing-Template.md:16]:
```suggestion
Update `InitialDatabaseSchema::MOST_RECENT_MIGRATION_NAME` to the latest migration name on production DB
Update the date on the file name of `yyyy_mm_dd_000000_initial_database_schema.php` 

```

File: .github/ISSUE_TEMPLATE/Standard-Migration-Squashing-Template.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+## Acceptance Criteria
+- [ ] All migrations created prior to the creation of this issue are squashed down into the initial_migration folder.
+
```
Comment (rashidsaf) [inline .github/ISSUE_TEMPLATE/Standard-Migration-Squashing-Template.md:19]:
```suggestion
- [ ] Latest migration should be updated on the codebase
- [ ] The init_schema file should be renamed to reflect today's date
- [ ] Most recent production DB  schema dump and the local DB schema should be exactly the same after squash

```

**Review by rashidsaf — CHANGES_REQUESTED** [review]

File: .github/ISSUE_TEMPLATE/Standard-Migration-Squashing-Template.md [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+## Negative Impact:
+This causes QA script, and general project initialization to slow down costing us developer hours
+
+# Proposed Solution:
+Squash migrations down into the init_schema directory of the migrations directory. 
+Create new migration classes as necessary.
+
+## Acceptance Criteria
+- [ ] All migrations created prior to the creation of this issue are squashed down into the initial_migration folder.
```
Comment (Chekote) [inline .github/ISSUE_TEMPLATE/Standard-Migration-Squashing-Template.md:18]:
```suggestion
- [ ] Production schema is NOT modified in any way
- [ ] Production migrations table is updated to reflect this new, single flattened migration
```

File: .github/ISSUE_TEMPLATE/Standard-Migration-Squashing-Template.md [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+
+## Announcement Message
+devchat: All migrations have been squashed, please rebase to master to have a consistent project initialization.
```
Comment (Chekote) [inline .github/ISSUE_TEMPLATE/Standard-Migration-Squashing-Template.md:21]: Rebase or merge. Devs should not rebase after code review has began.

**Review by Chekote — CHANGES_REQUESTED** [review]

**Review by PayneWheat — CHANGES_REQUESTED** [review]
I haven't squashed the migrations before, but looks like Rashid and Donald suggested some valuable info.

**Review by rashidsaf — APPROVED** [review]

**Review by Chekote — APPROVED** [review]

**Review by PayneWheat — APPROVED** [review]

**Pull request merged**